### ISSUE TITLE
Invalid RsaKeyAlgorithm publicExponent returned

### ISSUE THREAD

**Issue by panva** [cloudflare/workerd#48]
```js
crypto.subtle.generateKey({
  name: 'RSA-PSS',
  hash: { name: 'SHA-256' },
  modulusLength: 2048,
  publicExponent: new Uint8Array([0x01, 0x00, 0x01]),
}, false, ['sign', 'verify'])
```

An RSA-PSS, RSASSA-PKCS1-v1_5, or RSA-OAEP crypto key that's generated or imported is supposed to have an `algorithm.publicExponent` property that's supposed to be a Uint8Array. [ref](https://w3c.github.io/webcrypto/#RsaKeyAlgorithm-dictionary).

Workerd (and most likely also the workers platform) returns an ArrayBuffer.

cc @jasnell

**Comment by panva** [comment]
ping @jasnell

**Comment by jasnell** [comment]
ah right, yeah forgot about this one. Making a ticket internally to make sure we remember!

**Comment by panva** [comment]
Fixed in #1268
